src/components/course/data/hooks.tsx

```diff
@@ -358,58 +367,66 @@ export const useCourseEnrollmentUrl = ({
     () => ({
       next: `${config.LMS_BASE_URL}/courses/${courseRunKey}/course`,
       // Redirect back to the same page with a failure query param
-      failure_url: `${global.location.origin}${location.pathname}?${baseQueryParams.toString()}`,
+      failure_url: `${global.location.origin}${
+        location.pathname
+      }?${baseQueryParams.toString()}`,
     }),
-    [config.LMS_BASE_URL, courseRunKey, baseQueryParams, location.pathname],
+    [config.LMS_BASE_URL, courseRunKey, baseQueryParams, location.pathname]
   );
 
   // TODO: use the new helper functions (createEnrollWithLicenseUrl, createEnrollWithCouponCodeUrl) to generate url
-  const enrollmentUrl = useMemo(
-    () => {
-      if (userSubsidyApplicableToCourse?.subsidyType === LICENSE_SUBSIDY_TYPE) {
-        const queryParams = new URLSearchParams({
-          ...baseEnrollmentOptions,
-          license_uuid: userSubsidyApplicableToCourse.subsidyId,
-          course_id: courseRunKey,
-          enterprise_customer_uuid: enterpriseConfig.uuid,
-          // We don't want any sidebar text we show the data consent page from this workflow since
-          // the text on the sidebar is used when a learner is coming from their employer's system.
-          left_sidebar_text_override: '',
-          source: 'enterprise-learner-portal',
-        });
-        return `${config.LMS_BASE_URL}/enterprise/grant_data_sharing_permissions/?${queryParams.toString()}`;
-      }
-
-      if (!sku) {
-        // No product SKU is present, so the course cannot be enrolled in.
-        return null;
-      }
-
+  const enrollmentUrl = useMemo(() => {
+    if (userSubsidyApplicableToCourse?.subsidyType === LICENSE_SUBSIDY_TYPE) {
       const queryParams = new URLSearchParams({
         ...baseEnrollmentOptions,
-        sku,
-        consent_url_param_string: `failure_url=${encodeURIComponent(global.location.href)}?${baseQueryParams.toString()}&left_sidebar_text_override=`,
+        license_uuid: userSubsidyApplicableToCourse.subsidyId,
+        course_id: courseRunKey,
+        enterprise_customer_uuid: enterpriseConfig.uuid,
+        // We don't want any sidebar text we show the data consent page from this workflow since
+        // the text on the sidebar is used when a learner is coming from their employer's system.
+        left_sidebar_text_override: "",
+        source: "enterprise-learner-portal",
       });
+      return `${
+        config.LMS_BASE_URL
+      }/enterprise/grant_data_sharing_permissions/?${queryParams.toString()}`;
+    }
```

Comment:
nit: I realize many of the formatting changes like this one are generated by your use of Prettier; that said, I disagree with Prettier's opinion here. Changes like this to go from 1 to 3 lines is unnecessary given our ESLint config allows lines up to 120 characters.